frontend/src/app/render_wasm/shape.cljs

```diff
@@ -101,20 +118,57 @@
       :fills      (api/set-shape-fills v)
       :blend-mode (api/set-shape-blend-mode v)
       :shapes     (api/set-shape-children v)
+
```

Comment:
Should we leave this comment here without a TODO or something similar?

Reply:
It is ok for now leave this, is a prototype and we will iterate on it